### PR TITLE
Recover

### PR DESCRIPTION
**Pull request by v01dstar**
Signed-off-by: v01dstar <yang.zhang@pingcap.com>

### What problem does this PR solve?

TL;DR:  Adding unsafe-recover commands as part of https://github.com/tikv/tikv/issues/10483

### Check List

Tests

- Unit test

Code changes

- Has HTTP API interfaces change 

Side effects: N/A

Related changes: This is the first PR for this project

### Release note

None

### CODE REVIEW

File: server/api/unsafe_operation.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 TiKV Project Authors.
```
Comment (Connor1996) [inline server/api/unsafe_operation.go:1]:
```suggestion
// Copyright 2021 TiKV Project Authors.
```

File: server/api/unsafe_operation_test.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
+// Copyright 2017 TiKV Project Authors.
```
Comment (Connor1996) [inline server/api/unsafe_operation_test.go:1]:
```suggestion
// Copyright 2021 TiKV Project Authors.
```

File: server/tso/global_allocator.go [hunk 3]
```diff
@@ -171,19 +171,20 @@ func (gta *GlobalTSOAllocator) GenerateTSO(count uint32) (pdpb.Timestamp, error)
...
 	defer cancel()
-	var (
-		globalTSOResp          pdpb.Timestamp  = pdpb.Timestamp{}
-		estimatedMaxTSO        *pdpb.Timestamp = &pdpb.Timestamp{}
-		suffixBits             int             = gta.allocatorManager.GetSuffixBits()
-		shouldRetry, skipCheck bool
-		err                    error
-	)
 	for i := 0; i < maxRetryCount; i++ {
+		var (
```
Comment (Connor1996) [inline server/tso/global_allocator.go:175]: What are these changes for

File: tests/pdctl/unsafe/unsafe_operation_test.go [hunk 4]
```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 TiKV Project Authors.
```
Comment (Connor1996) [inline tests/pdctl/unsafe/unsafe_operation_test.go:1]:
```suggestion
// Copyright 2021 TiKV Project Authors.
```

**Review by Connor1996 — COMMENTED** [review]
rest LGTM

File: server/tso/global_allocator.go (hunk 3 above)
Comment (v01dstar) [reply server/tso/global_allocator.go]: This is due to outdated base branch I guess, updated.

**Pull request merged**